### PR TITLE
 Implementation of Dijkstra's Algorithm

### PR DESCRIPTION
**Pull request by yashaswibiyahut**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description
 Implementation of Dijkstra's Algorithm
Issue no -#47

### Checklist

- [x]  I've read the contribution guidelines.

- [x]  I've checked the issue list before deciding what to submit.

- [x]  I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by yashaswibiyahut** [comment]
kindly review this PR @Kajol-Kumari @SKAUL05 @siddharth25pandey @iamrajiv @HarshCasper

File: C/ds/Dijsktra.c [hunk 1]
```diff
@@ -0,0 +1,119 @@
+/*	Dijkstra's Algorithm and code in C    */
+#include<stdio.h>
+#include<stdlib.h>
+
+void main(){
+
+	int cost[10][10],distance[10],path[10][10],n,v,p,row,column,min,index=1,i,j;
```
Comment (siddharth25pandey) [inline C/ds/Dijsktra.c]: Use Dynamic Values

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@yashaswibiyahut Do the Suggested Changes

File: C/ds/Dijsktra.c (hunk 1 above)
Comment (yashaswibiyahut) [reply C/ds/Dijsktra.c]: done @siddharth25pandey 
kindly review it

File: C/ds/Dijsktra.c [hunk 2]
```diff
@@ -3,8 +3,11 @@
 #include<stdlib.h>
 
 void main(){
-
-	int cost[10][10],distance[10],path[10][10],n,v,p,row,column,min,index=1,i,j;
+	
+	int size;
+	printf("Enter the size of array:");
```
Comment (siddharth25pandey) [inline C/ds/Dijsktra.c]: User will not understand what is mean by size of array 
Please display the message clearly, what are you taking in form of **size**

Comment (siddharth25pandey) [reply C/ds/Dijsktra.c]: Display some relevant message

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@yashaswibiyahut Please do the modifications as required

**Review by siddharth25pandey — APPROVED** [review]
@yashaswibiyahut Now it's perfect
Able to merge
Thanks!

File: C/ds/Dijsktra.c (hunk 2 above)
Comment (yashaswibiyahut) [reply C/ds/Dijsktra.c]: done @siddharth25pandey 
kindly review it

**Comment by yashaswibiyahut** [comment]
@Kajol-Kumari @plazzy99 
kindly review and merge this PR

**Pull request closed without merging**